src/torrent/common.h

```diff
@@ -81,7 +82,6 @@ class Resolver;
 
 namespace tracker {
 class DhtController;
-class Manager;
```

Comment:
This is wrong, tracker::Manager is supposed to be there and torrent::Manager is internal to libtorrent.

Reply:
removed.